docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -21,25 +21,23 @@ Partitioning is another term for physically dividing large tables in YugabyteDB
 
 Since partitioned tables do not appear nor act differently from the original table, applications accessing the database are not always aware of the fact that partitioning has taken place.
 
-It is recommended to use partitioned tables with a query planner that can decide which tables to access and which ones to ignore based on the table content. 
-
 YSQL supports the following types of partitioning:
 
 - Range partitioning, when a table is partitioned into ranges defined by one or more key columns. In this case, the ranges of values assigned to partitions do not overlap.
```

Comment:
Actually we should specify HASH partitioning here, like PostgreSQL does, maybe we can skip examples and all, but its good to mention it here. Otherwise it gives the impression that although we are using a fork of Postgres we are not supporting hash partitioning. 
https://www.postgresql.org/docs/11/ddl-partitioning.html

Reply:
Added hash to the list